parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -78,13 +78,12 @@ installGPUDrivers() {
   wait_for_apt_locks
   retrycmd_if_failure_no_stats 120 5 25 cat $GPU_DEST/tmp/nvidia-docker.list >/etc/apt/sources.list.d/nvidia-docker.list || exit 85
   apt_get_update
-  retrycmd_if_failure 30 5 3600 apt-get install -y linux-headers-$(uname -r) gcc make dkms || exit 85
```

Comment:
This function is always executed when `installDeps` is executed, so there's no need to have two separate `apt-get install` implementations (and this was wasn't being retried).